### PR TITLE
NMA-1029: use dashpay sync system for Dash Wallet (headers first)

### PR DESCRIPTION
**Pull request by HashEngineering**
## Related PR's and Dependencies

https://github.com/dashevo/dashj/pull/127
## How Has This Been Tested?

- [x] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

**Review by Syn-McJ — APPROVED** [review]
Looks good

File: wallet/src/de/schildbach/wallet/ui/InputParser.java [hunk 1]
```diff
@@ -91,7 +90,7 @@ public StringInputParser(final String input, boolean supportAnypayUrls) {
 
         @Override
         public void parse() {
-            if (input.startsWith(CoinDefinition.coinURIScheme.toUpperCase() + ":-")) {
+            if (input.startsWith("DASH:-")) {
```
Comment (Syn-McJ) [inline wallet/src/de/schildbach/wallet/ui/InputParser.java:93]: If we have `SendCoinsActivity.DASH_SCHEME` constant, shouldn't we use it here?

**Review by Syn-McJ — APPROVED** [review]
Looks good, might make some possible improvements with string constants

File: wallet/src/de/schildbach/wallet/ui/InputParser.java (hunk 1 above)
Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/ui/InputParser.java:93]: Yes, or from something in DashJ.  Will make this change.

**Pull request merged**